model/Human-GEM.yml

```
@@ -162317,10 +162301,10 @@
       - gene_reaction_rule: "ENSG00000121691 or ENSG00000166016"
       - rxnNotes: ""
       - rxnFrom: "HMRdatabase"
-      - eccodes: "1.11.1.6"
+      - eccodes: "1.11.1.6;1.11.1.21"
```

```suggestion
      - eccodes: "1.11.1.6"
```
[EC1.11.1.6](https://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-1.11.1.6) is the canonical catalase activity that is also meant here. [EC1.11.1.21](https://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-1.11.1.21) a less-specific donor:hydrogen-peroxide oxidoreductase, which can use H2O2 as donor., found in [bacteria and fungi](https://www.brenda-enzymes.org/enzyme.php?ecno=1.11.1.21).

---

Oh whoops I missed what the difference between those buttons was; I'll take care of that